.github/workflows/merge-upstream-prometheus.yml

```
@@ -244,60 +222,62 @@ jobs:
+            ## Merge Conflicts Detected
+            
+            *This PR was automatically created by the [merge-upstream-prometheus workflow](https://github.com/grafana/mimir-prometheus/blob/main/.github/workflows/merge-upstream-prometheus.yml) due to merge conflicts.*
             
             ### Details
             - **Upstream branch**: `${{ needs.check-merge.outputs.upstream_branch }}`
             - **Local branch**: `${{ needs.check-merge.outputs.local_branch }}`
             - **Latest upstream commit**: ${{ format('[`{0}`](https://github.com/prometheus/prometheus/commit/{0})', needs.check-merge.outputs.upstream_commit) }}
             - **Latest upstream commit date**: ${{ needs.check-merge.outputs.commit_date }}
             
-            ## Merge Conflicts Detected
-            
-            *This DRAFT PR was automatically created by the [merge-upstream-prometheus workflow](https://github.com/grafana/mimir-prometheus/blob/main/.github/workflows/merge-upstream-prometheus.yml) due to merge conflicts.*
-            
             ### Action Required
-            This PR contains **unresolved merge conflicts** with conflict markers preserved in the files. To complete this merge:
+            This **closed PR** serves as a placeholder that holds the branch and instructions for conflict resolution. Follow these steps to resolve conflicts and reopen this PR:
             
             ```bash
-            # Fetch and check out this branch locally
-            git fetch origin ${{ needs.check-merge.outputs.local_branch }}-merge-conflicts-${{ needs.check-merge.outputs.timestamp }}
-            git checkout ${{ needs.check-merge.outputs.local_branch }}-merge-conflicts-${{ needs.check-merge.outputs.timestamp }}
+            # 1. Fetch and check out the empty branch created by CI
+            git fetch origin
+            git checkout ${{ needs.check-merge.outputs.bot_branch }}
+            
+            # 2. Merge the upstream commit to trigger conflicts
+            git merge ${{ needs.check-merge.outputs.upstream_commit }} --no-edit
             
-            # Edit each conflicted file and resolve conflicts
-            # Look for these markers and resolve:
-            # <<<<<<< HEAD
-            # (your local changes)
-            # =======
-            # (upstream changes)  
-            # >>>>>>> commit_hash
+            # 3. If conflicts occur:
+            #    - Edit conflicted files and resolve conflicts
+            #    - Look for conflict markers: <<<<<<< HEAD, =======, >>>>>>>
+            #    - Remove conflict markers after resolving
+            #    - Run: git add . && git merge --continue
             
-            # After resolving all conflicts, commit the changes
-            git add .
-            git commit -m "Resolve merge conflicts"
+            # 4. Push your resolved merge (no force-push needed)
```

```suggestion
            # 4. Push your resolved merge (no force-push needed)

            # 5. Re-open the closed PR.
```

Unless pushing somehow auto-reopens the PR? :thinking: